roles/configure-nexus/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: "Look up Nexus URL"
+  command: >
+    oc get route {{ nexus_name | default('nexus') }} -n {{ nexus_project }} -o custom-columns=ROUTE:.spec.host --no-headers
```

Comment:
`nexus_project` as a variable here is problematic when we need to make project names unique. I tested that `content.namespace` is set and usable here. It's a bit of a leaky abstraction, but it will make life much easier if used here. I dont think this role needs to be portable, and if it does, then we can have logic to set the variable.